fdbserver/storageserver.actor.cpp

```diff
@@ -1642,6 +1651,8 @@ ACTOR Future<GetKeyValuesReply> readRange(StorageServer* data,
 
 			int prevSize = result.data.size();
 			merge(result.arena,
+			      result.arena,
```

Comment:
Why do you pass `result.arena` twice?

Reply:
I wanted to make the merge function generic enough such that you could pass an arena for the output and both inputs. In this case we the ptree and output share the same arena so that's why i pass it twice. I can make it more specific to range reads but not sure if that's what we want.